js/index.js

```diff
@@ -137,6 +138,10 @@ function buildQuickTryUI_v1_0() {
         if (config[supported_apps[0]].readme?.text) {
             markdown_payload_url = config[supported_apps[0]].readme.text;
         }
+
+        if (config[supported_apps[0]].console_baudrate) {
+            consoleBaudrateFromToml = config[supported_apps[0]].console_baudrate;
```

Comment:
In case of normal launchpad which has multiple apps, it may not be the only app so referencing just [0]  element may not be right

Reply:
This code is for the initial app which will load at the start, whenever user change the app from dropdown the console_baudrate also changes as per app which is handled at callback function given to **$('#device').on('change')** as follows:
```
    if (config[deviceTypeSelect.value].console_baudrate) {
        consoleBaudrateFromToml = config[deviceTypeSelect.value].console_baudrate;
    }
```